code/modules/projectiles/guns/projectile.dm

```diff
@@ -24,6 +24,7 @@
 	//For MAGAZINE guns
 	var/magazine_type = null	//the type of magazine that the gun comes preloaded with
 	var/obj/item/ammo_magazine/ammo_magazine = null //stored magazine
+	var/allowed_magazines		//determines list of which magazines will fit in the gun

```

Comment:
Is this var used anymore? Because at the moment, it is not, beyond the definition in automatic/tommygun.

Can you test whether or not the tommygun can be loaded with both magazines if you remove them?